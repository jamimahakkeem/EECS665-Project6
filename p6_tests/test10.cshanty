int main() {
	int a;
	a = 0;
	if(a < 1) {
		a = 1;
	}

	bool b;
	b = true;

	return 0;
}