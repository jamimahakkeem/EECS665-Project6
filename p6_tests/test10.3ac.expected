[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN func LOCALS]
a (local var of 8 bytes)
tmp0 (tmp var of 8 bytes)
[END func LOCALS]
fun_func:   enter func
            RECEIVE [a]
            [tmp0] := [a] EQ64 0
            IFZ [tmp0] GOTO lbl_1
            [a] := [a] ADD64 1
lbl_1:      nop
lbl_0:      leave func
